contrib/nydusify/pkg/converter/provider/provider.go

```diff
@@ -34,6 +36,14 @@ import (
 
 var LayerConcurrentLimit = 5
 
+// Media type constants for container image formats
+const (
+	// Docker manifest list media type (legacy)
+	DockerManifestListMediaType = "application/vnd.docker.distribution.manifest.list.v2+json"
+	// Docker manifest media type (legacy)
+	DockerManifestMediaType = "application/vnd.docker.distribution.manifest.v2+json"
```

Comment:
These two media type has been defined in `github.com/containerd/containerd/images`.